src/actions.ts

```diff
@@ -904,7 +904,16 @@ export const actions: {[k: string]: QueryHandler} = {
 		}
 		return {password: pw};
 	},
-
+	async 'replays/batch.json'(params) {
+		if (!params.ids) {
+			throw new ActionError("Invalid batch replay request, must provide ids");
+		}
+		const ids: string[] = params.ids.split(',');
```

Comment:
Probably this should be programatically capped in length here in addition to the query.